bpfman-api/src/config.rs

```diff
@@ -63,6 +63,15 @@ impl XdpMode {
             XdpMode::Hw => XdpFlags::HW_MODE,
         }
     }
+
+    pub fn from_u32(val: u32) -> Self {
```

Comment:
This should either be `impl TryFrom<u32>` or `impl From<u32>` - I'd lean towards the former to avoid the panic.

Reply:
Done!

Reply:
(I think XdpMode should maybe be moved to lib.rs in the future though)